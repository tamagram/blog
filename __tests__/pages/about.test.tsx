/**
 * @jest-environment jsdom
 */
import { getPage } from "next-page-tester";

describe("About", () => {
  test("Render About page", async () => {
    const { serverRenderToString } = await getPage({
      route: "/about",
    });
    expect(serverRenderToString().html).toMatchSnapshot();
  });
});
