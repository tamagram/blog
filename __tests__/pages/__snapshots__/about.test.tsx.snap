// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Render About page 1`] = `"<html data-reactroot=\\"\\"><head></head><body><div id=\\"__next\\"><div class=\\"layout\\"><header class=\\"header\\"><nav class=\\"header__nav\\"><li class=\\"header__nav__li\\"><a class=\\"header__nav__li__a\\" href=\\"/\\">Timeline</a></li><li class=\\"header__nav__li\\"><a class=\\"header__nav__li__selected_a\\" href=\\"/about\\">About</a></li></nav><div class=\\"header__div_menu\\"><a href=\\"#light_box\\"><span style=\\"box-sizing:border-box;display:inline-block;overflow:hidden;width:initial;height:initial;background:none;opacity:1;border:0;margin:0;padding:0;position:relative;max-width:100%\\"><span style=\\"box-sizing:border-box;display:block;width:initial;height:initial;background:none;opacity:1;border:0;margin:0;padding:0;max-width:100%\\"><img style=\\"display:block;max-width:100%;width:initial;height:initial;background:none;opacity:1;border:0;margin:0;padding:0\\" alt=\\"\\" aria-hidden=\\"true\\" src=\\"data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNDAiIGhlaWdodD0iNDAiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmVyc2lvbj0iMS4xIi8+\\"/></span><img alt=\\"menu\\" src=\\"data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7\\" decoding=\\"async\\" data-nimg=\\"intrinsic\\" style=\\"position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%\\"/><noscript><img alt=\\"menu\\" srcSet=\\"/_next/image?url=%2Fmenu.svg&amp;w=48&amp;q=75 1x, /_next/image?url=%2Fmenu.svg&amp;w=96&amp;q=75 2x\\" src=\\"/_next/image?url=%2Fmenu.svg&amp;w=96&amp;q=75\\" decoding=\\"async\\" data-nimg=\\"intrinsic\\" style=\\"position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%\\" loading=\\"lazy\\"/></noscript></span></a></div><div class=\\"header__div_light_box\\" id=\\"light_box\\"><div class=\\"header__div_light_box__div_menu_close\\"><a href=\\"#\\"><span style=\\"box-sizing:border-box;display:inline-block;overflow:hidden;width:initial;height:initial;background:none;opacity:1;border:0;margin:0;padding:0;position:relative;max-width:100%\\"><span style=\\"box-sizing:border-box;display:block;width:initial;height:initial;background:none;opacity:1;border:0;margin:0;padding:0;max-width:100%\\"><img style=\\"display:block;max-width:100%;width:initial;height:initial;background:none;opacity:1;border:0;margin:0;padding:0\\" alt=\\"\\" aria-hidden=\\"true\\" src=\\"data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNDAiIGhlaWdodD0iNDAiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmVyc2lvbj0iMS4xIi8+\\"/></span><img alt=\\"menu\\" src=\\"data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7\\" decoding=\\"async\\" data-nimg=\\"intrinsic\\" style=\\"position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%\\"/><noscript><img alt=\\"menu\\" srcSet=\\"/_next/image?url=%2Fmenu-close.svg&amp;w=48&amp;q=75 1x, /_next/image?url=%2Fmenu-close.svg&amp;w=96&amp;q=75 2x\\" src=\\"/_next/image?url=%2Fmenu-close.svg&amp;w=96&amp;q=75\\" decoding=\\"async\\" data-nimg=\\"intrinsic\\" style=\\"position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%\\" loading=\\"lazy\\"/></noscript></span></a></div><nav class=\\"header__div_light_box__nav\\"><li class=\\"header__div_light_box__nav__li\\"><a class=\\"header__div_light_box__nav__li__selected_a\\" href=\\"/\\">About</a></li><li class=\\"header__div_light_box__nav__li\\"><a class=\\"header__div_light_box__nav__li__a\\" href=\\"/blog\\">Blog</a></li></nav></div></header><main class=\\"main\\"><section class=\\"main__section__avatars\\"><div><span style=\\"box-sizing:border-box;display:inline-block;overflow:hidden;width:initial;height:initial;background:none;opacity:1;border:0;margin:0;padding:0;position:relative;max-width:100%\\"><span style=\\"box-sizing:border-box;display:block;width:initial;height:initial;background:none;opacity:1;border:0;margin:0;padding:0;max-width:100%\\"><img style=\\"display:block;max-width:100%;width:initial;height:initial;background:none;opacity:1;border:0;margin:0;padding:0\\" alt=\\"\\" aria-hidden=\\"true\\" src=\\"data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjAwIiBoZWlnaHQ9IjIwMCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4=\\"/></span><img alt=\\"tamagram\\" src=\\"data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7\\" decoding=\\"async\\" data-nimg=\\"intrinsic\\" class=\\"main__section__avatar_img\\" style=\\"position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%\\"/><noscript><img alt=\\"tamagram\\" srcSet=\\"/_next/image?url=%2Ftamagram.jpg&amp;w=256&amp;q=75 1x, /_next/image?url=%2Ftamagram.jpg&amp;w=640&amp;q=75 2x\\" src=\\"/_next/image?url=%2Ftamagram.jpg&amp;w=640&amp;q=75\\" decoding=\\"async\\" data-nimg=\\"intrinsic\\" style=\\"position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%\\" class=\\"main__section__avatar_img\\" loading=\\"lazy\\"/></noscript></span></div><div class=\\"main__section__avatar_name\\">tama</div><div class=\\"main__section__icons\\"><a href=\\"https://github.com/tamagram\\"><span style=\\"box-sizing:border-box;display:inline-block;overflow:hidden;width:initial;height:initial;background:none;opacity:1;border:0;margin:0;padding:0;position:relative;max-width:100%\\"><span style=\\"box-sizing:border-box;display:block;width:initial;height:initial;background:none;opacity:1;border:0;margin:0;padding:0;max-width:100%\\"><img style=\\"display:block;max-width:100%;width:initial;height:initial;background:none;opacity:1;border:0;margin:0;padding:0\\" alt=\\"\\" aria-hidden=\\"true\\" src=\\"data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNDAiIGhlaWdodD0iNDAiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmVyc2lvbj0iMS4xIi8+\\"/></span><img alt=\\"github\\" src=\\"data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7\\" decoding=\\"async\\" data-nimg=\\"intrinsic\\" style=\\"position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%\\"/><noscript><img alt=\\"github\\" srcSet=\\"/_next/image?url=%2Fgithub-icon.svg&amp;w=48&amp;q=75 1x, /_next/image?url=%2Fgithub-icon.svg&amp;w=96&amp;q=75 2x\\" src=\\"/_next/image?url=%2Fgithub-icon.svg&amp;w=96&amp;q=75\\" decoding=\\"async\\" data-nimg=\\"intrinsic\\" style=\\"position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%\\" loading=\\"lazy\\"/></noscript></span></a><a href=\\"https://twitter.com/tamagrm\\"><span style=\\"box-sizing:border-box;display:inline-block;overflow:hidden;width:initial;height:initial;background:none;opacity:1;border:0;margin:0;padding:0;position:relative;max-width:100%\\"><span style=\\"box-sizing:border-box;display:block;width:initial;height:initial;background:none;opacity:1;border:0;margin:0;padding:0;max-width:100%\\"><img style=\\"display:block;max-width:100%;width:initial;height:initial;background:none;opacity:1;border:0;margin:0;padding:0\\" alt=\\"\\" aria-hidden=\\"true\\" src=\\"data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNDAiIGhlaWdodD0iNDAiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmVyc2lvbj0iMS4xIi8+\\"/></span><img alt=\\"github\\" src=\\"data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7\\" decoding=\\"async\\" data-nimg=\\"intrinsic\\" style=\\"position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%\\"/><noscript><img alt=\\"github\\" srcSet=\\"/_next/image?url=%2Ftwitter-icon.svg&amp;w=48&amp;q=75 1x, /_next/image?url=%2Ftwitter-icon.svg&amp;w=96&amp;q=75 2x\\" src=\\"/_next/image?url=%2Ftwitter-icon.svg&amp;w=96&amp;q=75\\" decoding=\\"async\\" data-nimg=\\"intrinsic\\" style=\\"position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%\\" loading=\\"lazy\\"/></noscript></span></a><a href=\\"https://zenn.dev/tamagram\\"><span style=\\"box-sizing:border-box;display:inline-block;overflow:hidden;width:initial;height:initial;background:none;opacity:1;border:0;margin:0;padding:0;position:relative;max-width:100%\\"><span style=\\"box-sizing:border-box;display:block;width:initial;height:initial;background:none;opacity:1;border:0;margin:0;padding:0;max-width:100%\\"><img style=\\"display:block;max-width:100%;width:initial;height:initial;background:none;opacity:1;border:0;margin:0;padding:0\\" alt=\\"\\" aria-hidden=\\"true\\" src=\\"data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNDAiIGhlaWdodD0iNDAiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmVyc2lvbj0iMS4xIi8+\\"/></span><img alt=\\"github\\" src=\\"data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7\\" decoding=\\"async\\" data-nimg=\\"intrinsic\\" style=\\"position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%\\"/><noscript><img alt=\\"github\\" srcSet=\\"/_next/image?url=%2Fzenn-icon.svg&amp;w=48&amp;q=75 1x, /_next/image?url=%2Fzenn-icon.svg&amp;w=96&amp;q=75 2x\\" src=\\"/_next/image?url=%2Fzenn-icon.svg&amp;w=96&amp;q=75\\" decoding=\\"async\\" data-nimg=\\"intrinsic\\" style=\\"position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%\\" loading=\\"lazy\\"/></noscript></span></a><a href=\\"https://tamagram.hatenablog.com/\\"><span style=\\"box-sizing:border-box;display:inline-block;overflow:hidden;width:initial;height:initial;background:none;opacity:1;border:0;margin:0;padding:0;position:relative;max-width:100%\\"><span style=\\"box-sizing:border-box;display:block;width:initial;height:initial;background:none;opacity:1;border:0;margin:0;padding:0;max-width:100%\\"><img style=\\"display:block;max-width:100%;width:initial;height:initial;background:none;opacity:1;border:0;margin:0;padding:0\\" alt=\\"\\" aria-hidden=\\"true\\" src=\\"data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNDAiIGhlaWdodD0iNDAiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmVyc2lvbj0iMS4xIi8+\\"/></span><img alt=\\"github\\" src=\\"data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7\\" decoding=\\"async\\" data-nimg=\\"intrinsic\\" style=\\"position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%\\"/><noscript><img alt=\\"github\\" srcSet=\\"/_next/image?url=%2Fhatena-icon.svg&amp;w=48&amp;q=75 1x, /_next/image?url=%2Fhatena-icon.svg&amp;w=96&amp;q=75 2x\\" src=\\"/_next/image?url=%2Fhatena-icon.svg&amp;w=96&amp;q=75\\" decoding=\\"async\\" data-nimg=\\"intrinsic\\" style=\\"position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%\\" loading=\\"lazy\\"/></noscript></span></a></div></section><section class=\\"main__section__profiles\\"><div class=\\"main__section__profiles__div\\"><h2># Profile</h2><p>**Name** <br/>Masahiro Tajima <br/>**Occupation** <br/>Student <br/>**Club** <br/>IPFactory <br/>MakeIT <br/></p><h2># Interest</h2><p>**WebDev** <br/>React <br/>**Cloud** <br/>AWS <br/></p></div></section></main><footer class=\\"footer\\">© tamagram 2021</footer></div>;</div></body></html>"`;
